src/ui/server/controllers/root/thank-you/index.test.ts

```diff
@@ -0,0 +1,34 @@
+import { get } from './index';
+import { mockReq, mockRes } from '../../../test-mocks';
+import { Request, Response } from '../../../../types';
+
+describe('controllers/thanks.txt', () => {
+  let req: Request;
+  let res: Response;
+
+  beforeEach(() => {
+    req = mockReq();
+    res = mockRes();
+  });
+
+  describe('get', () => {
+    it('should render /thanks.txt file with text/plain content type', () => {
+      get(req, res);
+
+      expect(res.type).toHaveBeenCalledWith('text/plain');
+    });
+
+    it('should return /thanks.txt file with expected content', () => {
+      get(req, res);
+
+      expect(res.write).toHaveBeenCalledWith('# We would like to thank the following:');
```

Comment:
Not sure if this would work, not a blocker.

```suggestion
      expect(res.write).toHaveBeenCalledWith('# We would like to thank the following:\n');
```

Reply:
Good spot, missed it have added.